connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufCatalystDataConversionSuite.scala

```diff
@@ -35,18 +36,32 @@ class ProtobufCatalystDataConversionSuite
     with SharedSparkSession
     with ExpressionEvalHelper {
 
-  private def checkResult(
+  private val testFileDesc = testFile("protobuf/catalyst_types.desc").replace("file:/", "/")
```

Comment:
All the sbt builds and tests are passing in ci/cd, but tests fail on my mac. The reason is these descriptor files are copied to _target/scala-2.12test-classes/_ rather than to _target/scala-2.12test-classes/**protobuf**_ . 

@SandishKumarHN Do you know what might be going on? I tried the same without this PR and it works fine.

Reply:
@rangadi  correct I'm also seeing sbt test failure, in the spark build the spark-protobuf tests are not running. I have made the changes here https://github.com/apache/spark/pull/38212/files#diff-48c0ee97c53013d18d6bbae44648f7fab9af2e0bf5b0dc1ca761e18ec5c478f2 can you take this change so that PR build runs with sbt tests?

Reply:
If it enables tests and fails, how do we fix it? :)

Reply:
Before this PR tests are passing with sbt, debugging the sbt-protoc generated class issue.

Reply:
yeah, I don't know how this is related to protobuf generation. 
I updated build_and_test.xml.

Reply:
The change is related to this line. Copying resources should not have anything to do with protoc plugin. But 🤷‍♂️

```
  (Test / PB.protoSources) := Seq((Test / sourceDirectory).value / "resources/protobuf"),
```

Reply:
I pushed a fix. Anyway this is temporary. We will be generating these rather than copying from `resources` directory.

Reply:
Btw, is there a quick way to run only protobuf/tests using sbt? Each time I run, it compiles catalyst & sql modules and runs those tests too.

Reply:
@rangadi this is what I use

```
sbt
project protobuf
test
```

Reply:
I think it just compiles sql, core, and catalyst since we use classes from those modules in spark-protobuf, but does not run other modules tests.